src/cnaas_nms/api/app.py

```diff
@@ -91,6 +92,7 @@ def handle_error(self, e):
 # Firmware
 api.add_resource(FirmwareApi, '/firmware')
 api.add_resource(FirmwareImageApi, '/firmware/<string:filename>')
+api.add_resource(FirmwareUpgradeApi, '/upgrade')
```

Comment:
Maybe the path should be /firmware/upgrade so users don't think you are upgrading the actual cnaas-nms api?

Reply:
Done.